flutter/documentation/result-spec.md

```diff
@@ -0,0 +1,129 @@
+  * `value_name`: string. Value from `setting.value.name` that is passed to backend
+* `throughput`: floating point number. Present for all results
+* `accuracy`: floating point number. Present for all results, including `loadgen_mode=PerformanceOnly` runs  
+  When actual accuracy is not available value `-1` must be used
+* `min_duration_ms`: floating point number  
+  Value from `task.min_duration_ms` for this benchmark from selected tasks.pbtxt file.
+* `measured_duration_ms`: floating point number  
+  Actual duration of the benchmark in milliseconds for start to finish.
+* `min_samples`: integer number  
+  Value from `task.min_query_count` for this benchmark from selected tasks.pbtxt file.
+* `measured_samples`: integer number  
+  Actual number of samples evaluated during the benchmark
+* `datetime`: string  
+  Contains datetime of the moment when benchmark **started**  
+  Format is: `yyyy-MM-ddTHH:mm:ss.sTZD`
+* `backend_accelerator_name`: string  
+  Backend-defined string describing actual accelerator used during this benchmark.  
+  Should typically match `accelerator_desc` from the `backend_settings` map but may be different in case of accelerator fallback.
+
+## Environment info
+
+Info about environment the app is running in. May change when you update your OS, change device hardware, or use another device.
+
+* `os`: string enum  
+  Allowed values:
+  * `android`
+  * `ios`
+  * `windows`
+* `os_version`: string  
+  Must be obtained from environment
+* `manufacturer`: string. Manufacturer of the device  
+  On Windows systems value must be `Unknown`.  
+  On iOS systems value must be `Apple`.  
+  On Android systems value must represent actual manufacturer name obtained from system environment.
+* `model`: string. Manufacturer-defined model name  
+  On Windows systems value must be `Unknown PC`.  
+  On iOS systems value must be human-readable iPhone name. For example: `iPhone SE 2nd Gen`.  
+  On Android systems value must represent actual model name obtained from system environment.
+* `selected_backend`: map
+  * `filename`: string  
+    Actual filename of the backend
+  * `vendor_name`: string  
+    Vendor name reported by backend
+  * `name`: string  
+    Backend name reported by backend
+
+## Application build info
+
+Constant info for this build of the app. The only way to change it is to use a different version of the app.
+
+* `version`: string
+* `build_number`: string
+* `official_release_flag`: bool  
+  Indicates if the official release flag was set for this build
+* `dev_test_flag`: bool  
+  Indicated if development test flag was set for this build
+* `backend_list`: list of strings  
+  Must contain actual list of backends that are included into this version of the app.
+* `git_branch`: string
+* `git_commit`: string
+* `git_dirty_flag`: bool  
+  Indicates if there are any local changes compared to the commit specified in `git_commit`.
```

Comment:
These build info look good to me.

question for all, should we have backend specific version info like these (e.g., SNPE version, Samsung ENN SDK version, and MTK Neural Delegate version)

Reply:
We would need to get this info from somewhere in the build time.
SNPE version can be obtained from SNPE SDK folder name, but I don't know where we could automatically get versions for other backends.

Reply:
We have the flexibility to use SNPE or TFLite for different models run in a single execution. So, if we need to, we may have to add seperate delegate names for separate use case. e.g., on SDM865, we use TFLite GPU delegate for MobileBERT and SNPE for rest of the models.

Reply:
>we may have to add seperate delegate names for separate use case

This should be in benchmark-specific fields, not in build info.

We have `backend_accelerator_name` field there. I think it should suffice for this.